doc/ref_model/chapters/chapter05.rst

```diff
@@ -169,6 +169,72 @@ Tracing                                Jaeger
 
 **Table 5-7a:** Service examples.
 
+Platform Services - Monitoring Service Requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The table below specifies a set of requirements for the Monitoring service (aka monitoring system).
+
++-------------+-----------------------------------------------------------------------+-------------------------------------------------------+
+| Reference   | Requirement                                                           | Notes                                                 |
++=============+=======================================================================+=======================================================+
+| pas.mon.001 | The Monitoring service must be able to collect data generated by or   | Capabilities to monitor applications, services,       |  
+|             | collected from any resource (physical and virtual infrastructure,     | operating systems, network protocols, system metrics  |
+|             | application, network, etc.)                                           | and infrastructure components                         |
++-------------+-----------------------------------------------------------------------+-------------------------------------------------------+
+| pas.mon.002 | The Monitoring service must be able to aggregate collected data       |                                                       |
++-------------+-----------------------------------------------------------------------+-------------------------------------------------------+
+| pas.mon.003 | The Monitoring service must be able to correlate data from different  |                                                       |
+|             | systems                                                               |                                                       |
++-------------+-----------------------------------------------------------------------+-------------------------------------------------------+
+| pas.mon.004 | The Monitoring service must at least perform active or passive        |                                                       |
+|             | monitoring                                                            |                                                       |
```

Comment:
Clarify wording and relation to 017:

```suggestion
| pas.mon.004 | The Monitoring service must be able to perform at least one           |                                                       |
|             | of active or passive monitoring                                       |                                                       |
```

"At least" already implies that supporting both is something optional.